tests/spec/conftest.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+
+from databuilder.query_language import Dataset
+
+
+@pytest.fixture
+def spec_test(in_memory_engine):
+    def run_test(table_data, series, expected_results):
```

Comment:
I disagree.  To me, there are two good reasons for using keywords: to differentiate between two parameters of the same type, and so that readers of the code can avoid diving into the implementation of a function to understand what it does.

Neither applies here, and in particular this isn't code that you're ever going to be "just passing through" -- if you're working on this code, you're going to need to understand exactly what `spec_test()` does, in which case you're going to be familiar enough with what the three parameters are that I don't think they need names.

Reply:
I've not got time to add keywords now, but won't object if somebody else does.